### ISSUE TITLE
load pre-trained model from checkpoint?

### ISSUE THREAD

**Issue by saya1984** [Kyubyong/dc_tts#28]
I'm trying to use the pre-trained model as a seed for a new voice, but I don't see any code that can load the pre-trained model from the checkpoint. Has anyone gotten that to work? 
Thank you

**Comment by noetits** [comment]
I think it is done implicitly in train.py by this line:
sv = tf.train.Supervisor(logdir=logdir, save_model_secs=0, global_step=g.global_step)

So if you put the pretrained model in the logdir, and set logdir in hyperparams.py correctly and continue training with your database, it should do what you want.

Noé

**Comment by saya1984** [comment]
Looks like you're right, thanks!
From the tf Supervisor page: "If the program crashes and is restarted, the managed session automatically reinitialize variables from the most recent checkpoint."

Of course now I have a new problem. I'll open a new issue as soon as I can articulate it clearly :)

**Comment by fathimad11** [comment]
Heyy anyone please help me, how to add the number of pretained model?
